contrib/seccomp-notify-plugin-rust/src/seccompwatcher.rs

```diff
@@ -0,0 +1,101 @@
+extern crate nc;
+
+mod mknod;
+
+use std::os::raw::c_char;
+
+pub const RUN_OCI_SECCOMP_NOTIFY_HANDLE_NOT_HANDLED: i32 = 0;
```

Comment:
could this be an enum instead?

Reply:
fixed!  I needed to add another function but it looks cleaner now.